README.md

```diff
@@ -277,7 +277,7 @@ API | Description | Auth | HTTPS | CORS |
 | [GitHub](https://docs.github.com/en/free-pro-team@latest/rest) | Make use of GitHub repositories, code and user info programmatically | `OAuth` | Yes | Yes |
 | [Gitlab](https://docs.gitlab.com/ee/api/) | Automate GitLab interaction programmatically | `OAuth` | Yes | Unknown |
 | [Gitter](https://developer.gitter.im/docs/welcome) | Chat for Developers | `OAuth` | Yes | Unknown |
-| [Host.io](https://host.io) | Domains Data API for Developers | `apiKey` | Yes | Yes |
```

Comment:
Thanks for resolving the conflict! 
Can I ask you to keep this record in README, please? It should be after your Hexabin API record.

Reply:
@marekdano Yes, I just added the record. Thank you!